assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -218,6 +268,13 @@ const baseResolvers = {
 		}
 		yield actions.receiveFeatureToursForView( viewTours, { viewContext } );
 	},
+
+	*getLastDismissedAt() {
+		const { value: lastDismissedAt } = yield Data.commonActions.await( getItem( 'feature_tour_last_dismissed_at' ) );
```

Comment:
The `await` action is neat 🙂

Reply:
Yeah, thanks to @aaemnnosttv for telling me about it - this was giving me grey hair 😂